components/dev-tools/easybuild/SPECS/easybuild.spec

```diff
@@ -66,7 +66,6 @@ cp %{_sourcedir}/*py .
 
 export EASYBUILD_BOOTSTRAP_SOURCEPATH=%{_sourcedir}
 export EASYBUILD_INSTALLPATH=%{install_path}
-export EASYBUILD_MODULE_SYNTAX=Tcl
```

Comment:
I included this change sort of as a reminder to self.

This controls the syntax used by EasyBuild to generate module files, in particular in the syntax of the EasyBuild module file that is generated as a part of the bootstrap procedure.

I'm not sure if matters at all, because the module file generated by EasyBuild is removed later (see the `rm %{version}` below, which is now likely going to fail since the module file generated by EasyBuild will now be `3.1.1.lua`).

The OHPC module file that is created is written in Tcl syntax however (see line 80 & onwards in this `.spec`).

Is that intentional?

Even though Lmod can consume module files in both Tcl and Lua, Lua should be preferred since that involves less overhead (Lmod translates Tcl module file to Lua on the fly via a `tcl2lua` tool every time a Tcl module file is consumed).

cc @crbaird